wsmatrix@martin.zurowietz.de/workspacePopup/workspaceManagerOverride.js

```diff
@@ -449,7 +449,7 @@ var WorkspaceManagerOverride = class {
                     });
 
                     let event = Clutter.get_current_event();
-                    let modifiers = event ? event.get_state() : 0;
+                    let modifiers = event ? event.get_state() & Clutter.ModifierType.MODIFIER_MASK : 0;
```

Comment:
If `MODIFIER_MASK` is used "to remove all reserved values", do we care about this at all? Could we just use `modifiers = 0` (if the popup timeout is 0)?

Reply:
If not, your solution seems to be fine and feel free to merge :wink:

Reply:
it will only remove the flags of the reserved modifiers, not all of them.
So basically it will remove all the modifiers from `MODIFIER_RESERVED_13_MASK` to `MODIFIER_RESERVED_29_MASK`